app/components/component-docs/dataDisplay/CoreChip.docs.js

```diff
@@ -30,306 +31,543 @@ function handleDelete() {
-    variant="outlined"
-  />
-</CoreStack>
-        `}
-        renderElement={
-          <CoreStack direction="row" spacing={1}>
-            <CoreChip
-              label="Custom delete icon"
-              onClick={handleClick}
-              onDelete={handleDelete}
-              deleteIcon={<CoreIcon icon="done" />}
-            />
-
-            <CoreChip
-              label="Custom delete icon"
-              onClick={handleClick}
-              onDelete={handleDelete}
-              deleteIcon={<CoreIcon icon="delete" />}
-              variant="outlined"
-            />
-          </CoreStack>
-        }
-      />
-
-      <CodeSample
-        title="Chip adornments"
-        description={
-          <>
-            <CoreTypographyBody1>
-          You can add ornaments to the beginning of the component.
-Use the avatar prop to add an avatar or use the icon prop to add an icon.
-            </CoreTypographyBody1>
-
-            <CoreH6>Avatar Chip</CoreH6>
-
-            <CoreTypographyBody1>
-  You can use the avatar prop to add an avatar to the Chip.
-            </CoreTypographyBody1>
-          </>
-
-        }
-        code={`<CoreStack direction="row" spacing={1}>
-  <CoreChip avatar={<CoreBox styleClasses={[CoreClasses.PADDING.PL1]}><CoreAvatar styleClasses={[CoreClasses.DATA_DISPLAY.AVATAR_SMALL]}>A</CoreAvatar></CoreBox>} label="Avatar" />
-  
-  <CoreChip
-    avatar={<CoreBox styleClasses={[CoreClasses.PADDING.PL1]}><CoreAvatar alt="Ananta" src="https://picsum.photos/200?random=1" styleClasses={[CoreClasses.DATA_DISPLAY.AVATAR_SMALL]} /></CoreBox>}
-    label="Avatar"
-    variant="outlined"
-  />
-</CoreStack>
-        `}
-        renderElement={
-          <CoreStack direction="row" spacing={1}>
-            <CoreChip avatar={<CoreBox styleClasses={[CoreClasses.PADDING.PL1]}><CoreAvatar styleClasses={[CoreClasses.DATA_DISPLAY.AVATAR_SMALL]}>A</CoreAvatar></CoreBox>} label="Avatar" />
+                <CoreChip
+                  label="Clickable Link"
+                  component="a"
+                  href="#basic-chip"
+                  variant="outlined"
+                  clickable
+                />
+              </CoreStack>
+            }
+          />
+
+          <CodeSample
+            title="  "
+            description={<>
+              <CoreH6> Custom delete icon </CoreH6>
+
+              <CoreTypographyBody1 styleClasses={[CoreClasses.MARGIN.MY1]}> <CodeBlock>deleteIcon</CodeBlock> prop can be used to provide a custom delete icon.
+              </CoreTypographyBody1></>}
+            code={`<CoreChip
+  label="Custom delete icon"
+  onClick={handleClick}
+  onDelete={handleDelete}
+  deleteIcon={<CoreIcon icon="done" />}
+/>
+<CoreChip
+  label="Custom delete icon"
+  onClick={handleClick}
+  onDelete={handleDelete}
+  deleteIcon={<CoreIcon icon="delete" />}
+  variant="outlined"
+/>`}
+            expandedCode={`import { CoreChip, CoreStack, CoreIcon } from "@wrappid/core";
```

Comment:
coding documentation also needs to be beautifully crafted.
```
    expandedCode={`
import { CoreChip, CoreStack, CoreIcon } from "@wrappid/core";
function handleClick() {
  console.info("CoreChip clicked! 🎉");
}
function handleDelete() {
  console.info("Delete icon clicked! 🗑️");
}
export default function CustomDeleteIconCoreChip() {
  return (
    <>
      <CoreStack direction="row" spacing={1}>
        <CoreChip
          label="Custom delete icon"
          onClick={handleClick}
          onDelete={handleDelete}
          deleteIcon={<CoreIcon icon="done" />}
        />
        <CoreChip
          label="Custom delete icon"
          onClick={handleClick}
          onDelete={handleDelete}
          deleteIcon={<CoreIcon icon="delete" />}
          variant="outlined"
        />
      </CoreStack>
    </>
  );
}
    `}
```

Reply:
It is beautifully crafted.